### PR TITLE
OSDOSC-7864: Migrated Using Group Sync Operator with Okta and ROSA

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.13+`

Issue:
[OSDOCS-7864](https://issues.redhat.com/browse/OSDOCS-7864)

Link to docs preview:
- [Using Group Sync Operator with Okta and ROSA](https://65058--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/rosa-mobb-groupsync-okta)

QE review:
- [ ] QE has approved this change.

Additional information:
Migrated the "Using Group Sync Operator with Okta and ROSA" topic from the MOBB docs to the ROSA docs.

### CODE REVIEW

File: rosa_tutorials/rosa-mobb-groupsync-okta.adoc [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-18
+//---
+//date: '2023-07-25
+//title: Using Group Sync Operator with Okta and ROSA
+//authors:
+//  - Thatcher Hubbard
+//---
+
+This guide focuses on how to synchronize Identity Provider (IDP) groups and users after configuring authentication in OpenShift Cluster Manager (OCM).
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-groupsync-okta.adoc:18]:
```suggestion
This guide focuses on how to synchronize Identity Provider (IDP) groups and users after configuring authentication in Red Hat OpenShift Cluster Manager (OCM).
```

File: rosa_tutorials/rosa-mobb-groupsync-okta.adoc [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+To set up group synchronization from Okta to ROSA you must:
+
+. Define groups and assign users in Okta.
+. Install the Group Sync Operator from the OpenShift Operator Hub.
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-groupsync-okta.adoc:23]:
```suggestion
. Install the Group Sync Operator from the OpenShift OperatorHub.
```

File: rosa_tutorials/rosa-mobb-groupsync-okta.adoc [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+. Create user IDs to synchronize with ROSA if they do not already exist:
++
+image::okta-usr.png[Okta Users]
+
+. Assign the newly created users to the appropriate group:
++
+image::okta-assign.png[Okta add user to group]
+
+[id="rosa-mobb-groupsync-okta-install-group-sync"]
+== Install the Group Sync Operator from the OpenShift Operator Hub
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-groupsync-okta.adoc:46]:
```suggestion
== Install the Group Sync Operator from the OpenShift OperatorHub
```

File: rosa_tutorials/rosa-mobb-groupsync-okta.adoc [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+. In the OpenShift Operator Hub, select the **Group Sync Operator**:
++
+image::grp-sync-opr-hub.png[Group Sync in Operator Hub]
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-groupsync-okta.adoc:50]:
```suggestion
image::grp-sync-opr-hub.png[Group Sync in OperatorHub]
```

File: rosa_tutorials/rosa-mobb-groupsync-okta.adoc [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+. Create a new secret named `okta-group-sync` in the **group-sync-operator** namespace. This will contain the Okta API key that was just created.
+
+. Using the OpenShift CLI (`oc`), create the secret using the following format:
++
+[source,terminal]
+----
+$ oc create secret generic okta-api-token --from-literal='okta-api-token=${API_TOKEN}' -n group-sync-operator
+----
+
+. Obtain values from Okta for the AppId and URL. The AppId is the client ID under the application created to support OpenID for the OCP Cluster(s). The URL is the same as the one used to admin Okta, without the `-admin` in the first term and should look something like this:
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-groupsync-okta.adoc:85]:
```suggestion
. Obtain values from Okta for the AppId and URL. The AppId is the client ID under the application created to support OpenID for the {product-name-first} cluster(s). The URL is the same as the one used to admin Okta, without the `-admin` in the first term and should look something like this:
```

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
@EricPonvelle a few suggestions for now or later.

**Comment by michaelryanmcneill** [comment]
/hold

Adding the hold flag to prevent merge until we're able to discuss if we really want an unsupported community operator in the tutorials section when we have a fully workable solution using group claims that is supported and recommend.

**Review by michaelryanmcneill — COMMENTED** [review]
After discussion, we've determined that since there are no intentions to support the Group Sync Operator within OCP, this article is not suitable for migration to product documentation. Sorry for the unnecessary work @EricPonvelle.

**Pull request closed without merging**